Makefile

```diff
@@ -34,8 +34,8 @@ test:
 	npm run sonar-test
 
 .PHONY: test-unit
-test-unit:
-	npm run unit
+test-unit: clean
+	npm run coverage
```

Comment:
there is no coverage script in the package.json for extensions-web, so i would keep it as   npm run unit